config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  resources :pages
```

Comment:
move to "/account/" namespace + it has to be nested in companies resources. But for now it's OK

Reply:
+